### PR TITLE
add about-us post

### PR DESCRIPTION
**Pull request by francisco-core**
Adds an "about us" section linked on the footer. It includes the financial report

### CODE REVIEW

File: content/about/about-us.en.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Join us!"
+author: "Direction"
+image: "/img/PrivacyLxLogoWhite.svg"
```
Comment (anadahz) [inline content/about/about-us.en.md:4]: This image is not visible.

File: content/about/about-us.pt.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Sobre a PrivacyLx"
+author: "Direção"
+image: "/img/PrivacyLxLogoWhite.svg"
```
Comment (anadahz) [inline content/about/about-us.pt.md:4]: Similarly, this image is not visible in post.

**Comment by anadahz** [comment]
@francisco-core Thank you for working on it. Apart from the post image that is not visible in the post, this PR looks good and can be merged.

**Comment by francisco-core** [comment]
green light to merge

**Comment by francisco-core** [comment]
it's already good enough

**Pull request merged**